grafana-dashboard-sync/export-and-commit

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+GITHUB_TOKEN=""
+API_KEY=""
```

Comment:
Grafana dashboard sync is not a part of provisioning because we use it only in mainnet. That's why I didn't make it a terraform template and I left these envs empty and one should put there proper values manually. Alternatively I can just use `$1`, `$2` and pass values as args.

Reply:
I would go with arguments or env variables passed externally.